examples/Example17/macros/example17.mac.in

```diff
@@ -136,8 +136,9 @@
 /example17/gun/print true
 
 #If false, the following parameters are ignored
-/example17/gun/randomizeGun false
-/example17/gun/addParticle e-
+/example17/gun/randomizeGun true
+#Usage: /example17/gun/addParticle [type] [weight] [energy]
+/example17/gun/addParticle e- -1 10 GeV
 /example17/gun/addParticle e+
```

Comment:
If weight is missing, I supposed you redistribute equally the remaining weight as we discussed, but if the energy is missing what is happening?

Reply:
It gets the energy set for the gun

Reply:
Updated, now it's mandatory to provide the energy